src/client/components/priceAndTodaysPriceChange/PriceAndTodaysPriceChange.jsx

```diff
@@ -5,26 +5,27 @@ import './PriceAndTodaysPriceChange.css';
 
 const PriceAndTodaysPriceChange = ({
   currentPrice,
+  isPositiveChange,
   priceChange,
   priceChangePercentage
 }) => {
-  const isPositiveChange = priceChange >= 0;
   return (
     <div className='priceAndTodaysPriceChange'>
-      <span className='currentPrice'>{ currentPrice }</span>
+      <div className='currentPrice'>{ currentPrice }</div>
       { isPositiveChange
```

Comment:
even though I think this was wrong before this PR - indent. also review my code better ;)

Reply:
But this is the child of `<div className='priceAndTodaysPriceChange'>`, so I don't think it should be indented

Reply:
you're right, my fault